var/spack/repos/builtin/packages/py-interval/package.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2013-2018 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyInterval(PythonPackage):
+    """Python interval and interval set implementation."""
+
+    homepage = "https://pypi.org/project/interval/1.0.0/"
+    url      = "https://files.pythonhosted.org/packages/b3/2d/b337afbd232ea1ea9c38401135054bf763e7930ea5e2e49bc39af35c3443/interval-1.0.0.tar.bz2"
```

Comment:
This is my only gripe here (apart from the tag, but I'll take a promise for a future change there). [URLs for Python packages should follow the official recommendation](https://spack.readthedocs.io/en/latest/build_systems/pythonpackage.html#url).

```suggestion
    url      = "https://pypi.io/packages/source/i/interval/interval-1.0.0.tar.bz2"
```

Reply:
I actually tried it before but not hard enough :) I had the wrong file extension...
Now fixed.